src/librustdoc/markdown.rs

```diff
@@ -51,8 +52,8 @@ crate fn render<P: AsRef<Path>>(
 
     let mut css = String::new();
     for name in &options.markdown_css {
-        let s = format!("<link rel=\"stylesheet\" type=\"text/css\" href=\"{name}\">\n");
-        css.push_str(&s)
+        // `write!`ing to a `String` is infallible
+        writeln!(css, r#"<link rel="stylesheet" type="text/css" href="{name}">"#).unwrap();
```

Comment:
Just just helped me realize that the `\n` is completely unneeded. Also, I think `write!` will work better (look below): no need for `unwrap`. So maybe something like:

```suggestion
        write!(&mut css, r#"<link rel="stylesheet" type="text/css" href="{name}">"#);
```

Reply:
Unfortunately, the compiler complains about an unused `Result` and the warning is promoted to an error.